core/src/test/java/apoc/export/json/ImportJsonEnterpriseTest.java

```diff
@@ -0,0 +1,96 @@
+package apoc.export.json;
+
+import apoc.util.CompressionAlgo;
+import apoc.util.Neo4jContainerExtension;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.neo4j.driver.Session;
+
+import java.io.File;
+
+import static apoc.ApocConfig.APOC_IMPORT_FILE_ENABLED;
+import static apoc.export.json.JsonImporter.MISSING_CONSTRAINT_ERROR_MSG;
+import static apoc.util.BinaryTestUtil.fileToBinary;
+import static apoc.util.CompressionConfig.COMPRESSION;
+import static apoc.util.TestContainerUtil.createEnterpriseDB;
+import static apoc.util.TestContainerUtil.testCall;
+import static apoc.util.Util.map;
+import static java.lang.String.format;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class ImportJsonEnterpriseTest {
```

Comment:
Why do we need an integration test instead of a unit test? Is it because we are touching the system database?

Reply:
I created a new enterprise test class because I had to add `"IS NODE KEY"` constraints which are not supported in `ImportJsonTest`